.github/workflows/scripts/translation_check.py

```
@@ -1,295 +1,274 @@
-    errors: dict[str, list[str]] = {}
-    for path in targets:
-        missing = check_file(path, valid_keys)
-        if missing:
-            errors[str(path)] = missing
-
-    if errors:
-        for file, tags in errors.items():
-            print(
-                f"File: {file}\n"
-                + "\n".join(f"  - Missing: {tag}" for tag in tags)
+        with open(path, "r", encoding="utf-8") as f:
```

Ruff (0.14.10)

Unnecessary mode argument

Remove mode argument

(UP015)